tests/test_causal_problem/test_graph_and_param.py

```diff
@@ -7,7 +8,9 @@
 from causalprog.graph import Graph
 
 
-def test_graph_and_parameter_interactions(graph: Graph) -> None:
+def test_graph_and_parameter_interactions(
+    two_normal_graph: Callable[..., Graph],
+) -> None:
     cp = CausalProblem(label="TestCP")
 
     # Without a graph, we can't do anything
```

Comment:
Can't comment on non-diff lines again, but I noticed that we can use the `raises_error` fixture here instead of the explicit `pytest.raises` block.